src/Symfony/Component/DependencyInjection/Argument/LocatorArgument.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection\Argument;
+
+use Symfony\Component\DependencyInjection\Reference;
+
+/**
+ * Represents a service locator able to lazy load a given range of services.
+ *
+ * @author Robin Chalas <robin.chalas@gmail.com>
+ */
+class LocatorArgument implements ArgumentInterface
+{
+    private $values;
+
+    /**
+     * @param Reference[] $values
```

Comment:
Given `ContainerInterface::get()` must return an object, scalars should be returned by `getParameter()`, right?